staging/src/k8s.io/kubectl/pkg/cmd/cp/cp.go

```diff
@@ -138,7 +137,7 @@ func extractFileSpec(arg string) (fileSpec, error) {
 	}
 
 	pod, file := arg[:i], arg[i+1:]
-	pieces := strings.Split(pod, "/")
+	pieces := strings.Split(pod, string(filepath.Separator))
```

Comment:
this is not right, the pod name is either `<pod>` or `<namespace>/<pod>`, so this should continue to split on "/"

See the documented example:
```
kubectl cp <some-namespace>/<some-pod>:/tmp/foo /tmp/bar
```

If a test did not catch this, then we need to add a test explicitly for this case.

Reply:
We have a test here: https://github.com/kubernetes/kubernetes/blob/28b46be97bb453be2e00eaf9e35ea1d6ecd41861/staging/src/k8s.io/kubectl/pkg/cmd/cp/cp_test.go#L55-L112 but since we're running that on a linux machine it works just fine.